src/pages/authpages/Login.js

```diff
@@ -0,0 +1,72 @@
+import "./Login.css";
+import {useState} from "react";
+import {Link,useNavigate,useLocation} from "react-router-dom";
+import {useAuth} from "../../contexts";
+import axios from "axios";
+
+
+function Login(){
+    
+    const { setUser } = useAuth();
+    const [email, setUserName] = useState("");
```

Comment:
Suggestion - Instead of using two different states for email and password, you can use the state which will be an object and add all connected data in one state object.